compute/cloud-client/instances/src/create_firewall_rule.php

```diff
@@ -77,13 +76,14 @@ function create_firewall_rule(string $projectId, string $firewallRuleName, strin
     //Create the firewall rule using Firewalls Client.
     $operation = $firewallsClient->insert($firewallResource, $projectId);
 
-    // Wait for the create operation to complete using a custom helper function.
-    // @see src/wait_for_operation.php
-    $operation = wait_for_operation($operation, $projectId);
-    if (empty($operation->getError())) {
+    // Wait for the operation to complete.
+    $operation->pollUntilComplete();
+    if ($operation->operationSucceeded()) {
+        // $result = $operation->getResult(); // Optionally get operation result
```

Comment:
For Compute LROs, `getResult` is always null. As far as I know there's no way to get one.

```suggestion
        // Do nothing! Operation succeeded
```

Reply:
Removed.